support/client/lib/mil-sym/cws.js

```diff
@@ -5288,10 +5338,78 @@ define( function(){
                   "symbolID": "G*GPOLKA--****X",
                   "tag": "DIRECTION OF ATTACK AVIATION",
                   "symbolType": "SimpleArrow"
+            },
+            "TACGRP.C2GM.SPL.ARA.AOO": {
+                  "symbolID": "G*GPSAO---****X",
+                  "tag": "AREA OF OPERATIONS",
+                  "symbolType": "MultiSegmentPolygon",
+                  "validModifiers": [ "uniqueDesignation1" ]
+            },
+            "TACGRP.C2GM.SPL.ARA.AHD": {
+                  "symbolID": "G*GPSAA---****X",
+                  "tag": "AIRHEAD",
+                  "symbolType": "MultiSegmentPolygon",
+                  "validModifiers": [ "uniqueDesignation1" ]
+            },
+            "TACGRP.C2GM.SPL.ARA.ENCMT": {
+                  "symbolID": "G*GPSAE---****X",
+                  "tag": "ENCIRCLEMENT",
+                  "symbolType": "MultiSegmentPolygon",
+                  "validModifiers": [ "hostile" ]
+            },
+            "TACGRP.C2GM.SPL.ARA.NAI": {
+                  "symbolID": "G*GPSAN---****X",
+                  "tag": "NAMED AREA OF INTEREST",
+                  "symbolType": "MultiSegmentPolygon",
+                  "validModifiers": [ "uniqueDesignation1" ]
+            },
+            "TACGRP.C2GM.OFF.LNE.AXSADV.AVN": {
+                  "symbolID": "G*GPOLAV--****X",
+                  "tag": "AVIATION AXIS OF ADVANCE",
+                  "symbolType": "MultiSegmentArrow",
+                  "arrowLeafOffset": 40
+            },
+            "TACGRP.C2GM.OFF.LNE.AXSADV.ABN": {
+                  "symbolID": "G*GPOLAA--****X",
+                  "tag": "AIRBORNE AXIS OF ADVANCE",
+                  "symbolType": "MultiSegmentArrow",
+                  "arrowLeafOffset": 40
+            },
+            "TACGRP.C2GM.OFF.LNE.AXSADV.ATK": {
+                  "symbolID": "G*GPOLAR--****X",
+                  "tag": "RWA ATTACK AXIS OF ADVANCE",
+                  "symbolType": "MultiSegmentArrow",
+                  "arrowLeafOffset": 40
+            },
+            "TACGRP.C2GM.OFF.LNE.AXSADV.GRD.MANATK": {
+                  "symbolID": "G*GPOLAGM-****X",
+                  "tag": "GROUND MAIN ATTACK",
+                  "symbolType": "MultiSegmentArrow",
+                  "arrowLeafOffset": 40
+            },
+            "TACGRP.C2GM.OFF.LNE.AXSADV.GRD.SUPATK": {
+                  "symbolID": "G*GPOLAGS-****X",
+                  "tag": "GROUND SUPPORTING ATTACK",
```

Comment:
This appears to repeat the name of a defined graphic.  Is one of them named incorrectly?

Reply:
Ah, good eyes.  It should be a multipoint arrow.